site/ko/alpha/tutorials/quickstart/advanced.ipynb

```diff
@@ -0,0 +1,390 @@
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "u48C9WQ774n4"
+   },
+   "outputs": [],
+   "source": [
+    "loss_object = tf.keras.losses.SparseCategoricalCrossentropy()\n",
+    "\n",
+    "optimizer = tf.keras.optimizers.Adam()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "JB6A1vcigsIe"
+   },
+   "source": [
+    "모델의 손실과 성능을 측정할 지표를 선택합니다. 에포크가 진행되는 동안 수집된 측정 지표를 바탕으로 최종 결과를 출력합니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "N0MqHFb4F_qn"
+   },
+   "outputs": [],
+   "source": [
+    "train_loss = tf.keras.metrics.Mean(name='train_loss')\n",
+    "train_accuracy = tf.keras.metrics.SparseCategoricalAccuracy(name='train_accuracy')\n",
+    "\n",
+    "test_loss = tf.keras.metrics.Mean(name='test_loss')\n",
+    "test_accuracy = tf.keras.metrics.SparseCategoricalAccuracy(name='test_accuracy')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "ix4mEL65on-w"
+   },
+   "source": [
+    "`tf.GradientTape`를 사용하여 모델을 훈련합니다:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "OZACiVqA8KQV"
+   },
+   "outputs": [],
+   "source": [
+    "@tf.function\n",
+    "def train_step(images, labels):\n",
+    "  with tf.GradientTape() as tape:\n",
+    "    predictions = model(images)\n",
+    "    loss = loss_object(labels, predictions)\n",
+    "  gradients = tape.gradient(loss, model.trainable_variables)\n",
+    "  optimizer.apply_gradients(zip(gradients, model.trainable_variables))\n",
+    "\n",
+    "  train_loss(loss)\n",
+    "  train_accuracy(labels, predictions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Z8YT7UmFgpjV"
+   },
+   "source": [
+    "이제 모델을 테스트합니다:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "xIKdEzHAJGt7"
+   },
+   "outputs": [],
+   "source": [
+    "@tf.function\n",
+    "def test_step(images, labels):\n",
+    "  predictions = model(images)\n",
+    "  t_loss = loss_object(labels, predictions)\n",
+    "\n",
+    "  test_loss(t_loss)\n",
+    "  test_accuracy(labels, predictions)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "i-2pkctU_Ci7"
+   },
+   "outputs": [],
+   "source": [
+    "EPOCHS = 5\n",
+    "\n",
+    "for epoch in range(EPOCHS):\n",
+    "  for images, labels in train_ds:\n",
+    "    train_step(images, labels)\n",
+    "\n",
+    "  for test_images, test_labels in test_ds:\n",
+    "    test_step(test_images, test_labels)\n",
+    "\n",
+    "  template = 'Epoch {}, Loss: {}, Accuracy: {}, Test Loss: {}, Test Accuracy: {}'\n",
```

Comment:
There is missing colon, Epoch {} → Epoch: {}
And I'd like to suggest to translate this line :)

Reply:
Good catch! :)